lib/av_verifier.ts

```diff
@@ -146,7 +146,8 @@ export class AVVerifier {
           const optionConfig = optionFinder(os.reference)
           return {
             reference: os.reference,
-            title: localizer(optionConfig.title)
+            title: localizer(optionConfig.title),
+            text: os.text,
```

Comment:
Nevermind, found it 😅

Reply:
You can make github hide whitespace changes, to make these kinds of things easier to spot:
![image](https://user-images.githubusercontent.com/100691050/224943311-c6baf107-ff2b-447f-a536-d304bba7e88d.png)